pytensor/scan/rewriting.py

```diff
@@ -1257,7 +1344,7 @@ def save_mem_new_scan(fgraph, node):
                 else:
                     # there is a **gotcha** here ! Namely, scan returns an
                     # array that contains the initial state of the output
-                    # as well. Which means that if have a initial state of
+                    # as well. Which means that if y have a initial state of
```

Comment:
```suggestion
                    # as well. Which means that if y has a initial state of
```

Reply:
Ill add the commit before merging to avoid re-running the CI